packages/constants/src/msf.ts

```diff
@@ -0,0 +1,273 @@
+export const EMPLOYMENT_TYPES = [
+  {id: '02', name: 'Temporary'},
+  {id: '03', name: 'Contract'},
+  {id: '05', name: 'Part Time'},
+  {id: '06', name: 'Freelance'},
+  {id: '07', name: 'Permanent'},
+  {id: '08', name: 'Full Time'},
+  {id: '09', name: 'Flexi Time'},
```

Comment:
I have checked MSF is actually returning as `Flexi Work` but our `api-profile` somehow put it as `Flexi Time` ^^"